app/retail/emails.py

```diff
@@ -932,8 +932,8 @@ def render_start_work_applicant_expired(interest, bounty):
 def render_new_bounty_roundup(to_email):
     from dashboard.models import Bounty
     from django.conf import settings
-    subject = "The Final Hours of Beyond Blockchain ⛓"
-    new_kudos_pks = [4144, 4141, 4143]
+    subject = "Grow Ethereum Draws Nearer"
```

Comment:
should we call it `Grow Ethereum Hackathon`?  i worry ppl wont know what 'Grow Ethereum' is unless we specify..

Reply:
good point. i'll update across the newsletter